std_msgs/Header header

bool is_shifting
bool allow_route
bool is_normal
bool start_trace
